.github/workflows/pr.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
     - name: Run chart-testing (list-changed)
       id: list-changed
       run: |
-        changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+        changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }} --excluded-charts sqnc-identity-service)
```

Comment:
we don't want to exclude it here, we want to exclude it in the job execution on paths-ignore
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore